gimp/pictonode-gimp-plugin/GIMP 2.99/pictonode/custom_nodes.py

```diff
@@ -473,6 +482,15 @@ def __init__(self, node_window, *args, **kwds) -> None:
     def remove(self, node):
         self.destroy()
 
+    def get_values(self):
+
+        ''' Returns dictionary of current state of custom values for the node'''
+
+        return {"opacity": self.opacity,
+                "x": self.x,
+                "y": self.y,
+                "scale": self.scale}
```

Comment:
I see that you used a dictionary here. Would it be at all possible to, instead of using the names of the properties, use the indices of the output links? It would make processing easier.

Reply:
Resolved in chat